sdk/documenttranslation/Azure.AI.DocumentTranslation/src/DocumentTranslationClient.cs

```diff
@@ -0,0 +1,495 @@
+            {
+                ResponseWithHeaders<DocumentTranslationSubmitBatchRequestHeaders> job = _serviceRestClient.SubmitBatchRequest(request, cancellationToken);
+                return new DocumentTranslationOperation(_serviceRestClient, _clientDiagnostics, job.Headers.OperationLocation);
+            }
+            catch (Exception e)
+            {
+                scope.Failed(e);
+                throw;
+            }
+        }
+
+        /// <summary>
+        /// Starts a translation operation
+        /// For document length limits, maximum batch size, and supported document formats, see
+        /// <a href="https://docs.microsoft.com/azure/cognitive-services/translator/document-translation/overview"/>.
+        /// </summary>
+        /// <param name="configurations">Sets the configurations for the translation operation
+        /// including source and target storage for documents to be translated. </param>
+        /// <param name="cancellationToken">A <see cref="CancellationToken"/> controlling the request lifetime.</param>
+        /// <exception cref="RequestFailedException">Service returned a non-success status code. </exception>
+        public virtual async Task<DocumentTranslationOperation> StartTranslationAsync(IEnumerable<TranslationConfiguration> configurations, CancellationToken cancellationToken = default)
+        {
+            var request = new BatchSubmissionRequest(configurations);
+            using DiagnosticScope scope = _clientDiagnostics.CreateScope($"{nameof(DocumentTranslationClient)}.{nameof(StartTranslationAsync)}");
+            scope.Start();
+
+            try
+            {
+                ResponseWithHeaders<DocumentTranslationSubmitBatchRequestHeaders> job = await _serviceRestClient.SubmitBatchRequestAsync(request, cancellationToken).ConfigureAwait(false);
+                return new DocumentTranslationOperation(_serviceRestClient, _clientDiagnostics, job.Headers.OperationLocation);
+            }
+            catch (Exception e)
+            {
+                scope.Failed(e);
+                throw;
+            }
+        }
+
+        /// <summary>
+        /// Starts a translation operation for documents in an Azure Blob Container.
+        /// For document length limits, maximum batch size, and supported document formats, see
+        /// <a href="https://docs.microsoft.com/azure/cognitive-services/translator/document-translation/overview"/>.
+        /// </summary>
+        /// <param name="sourceBlobContainerSas">The SAS URL for the source container containing documents to be translated. </param>
+        /// <param name="targetBlobContainerSas">The SAS URL for the target container to which the translated documents will be written. </param>
+        /// <param name="targetLanguageCode">Language code to translate documents to. For supported languages see
+        /// <a href="https://docs.microsoft.com/azure/cognitive-services/translator/language-support#translate"/>.</param>
```

Comment:
should we point them to our methods instead of the documentation?

Reply:
same comment for glossaries

Reply:
I think we could do it for glossaries

Reply:
I pointed to the glossaries method instead of docs